addons/default-storage-class/storage-class.yaml

```diff
@@ -206,3 +206,17 @@ metadata:
 driver: dobs.csi.digitalocean.com
 deletionPolicy: Delete
 {{ end }}
+
+{{ if eq .Config.CloudProvider.CloudProviderName "vmwareCloudDirector" }}
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+  name: vcd-disk-dev
+provisioner: named-disk.csi.cloud-director.vmware.com
+reclaimPolicy: Delete
+parameters:
+  storageProfile: {{ default "*" .Params.storageProfile | quote }}
```

Comment:
@kron4eg if a valid value is not provided then this would be non-functional. How about moving `storageProfile` to https://github.com/kubermatic/kubeone/blob/master/pkg/apis/kubeone/v1beta2/types.go#L287 and make it a required field?